docs-2.0/2.quick-start/0.FAQ.md

```diff
@@ -0,0 +1,92 @@
+      | vid          | age |
+      | 'Tim Duncan' | 42  |
+    And no side effects
+```
+
+The keywords in the preceding example are described as follows:
+
+|Keyword|Description|
+|-|-|
+|`Feature`|Describes the topic of the current `.feature` file.|
+|`Background`|Describes the background information of the current `.feature` file.|
+|`Given`|Describes the prerequisites of running the test statements in the current `.feature` file.|
+|`Scenario`|Describes the purpose of the scenario. If there is the `@skip` before `Scenario`, this scenario may not work and don't use it as a working example.|
+|`When`|Describes the nGQL statement to be executed.|
+|`Then`|Describes the expected result of running the statement in the `When` clause. If the result in your environment does not match the result described in the `.feature` file, submit an [issue](https://github.com/vesoft-inc/nebula-graph/issues) to inform the Nebula Graph team.|
+|`And`|Describes the side effects of running the statement in the `When` clause.|
+
+Welcome to [add more practical scenarios](https://docs.nebula-graph.io/1.1/manual-EN/4.contributions/how-to-contribute/) and become a Nebula Graph contributor.
+
+### Is nGQL compatible with openCypher 9?
+
+nGQL is partially compatible with openCypher 9. Known incompatible items are listed in [Nebula Graph Issues](https://github.com/vesoft-inc/nebula-graph/issues). Submit an issue with the `incompatible` tag if you find a new issue of this type.
+
+The Nebula Graph team is striving for higher compatibility with openCypher 9.
+
+The following are some major difference (incompatible by design) between nGQL and openCypher 9.
+
+| nGQL | openCypher 9 |
+| - | - |
+| strong schema | schema optional |
+| equality operator '==' |  equality operator '=' |
+
+almost all DMLs (CREATE, MERGE, etc).
+
+> Please also note that [openCypher](http://www.opencypher.org/) 9 and [Cypher](https://neo4j.com/developer/cypher/) have some difference (in grammar and licence). 
+
+> E.g., Cypher requires `All Cypher statements are explicitly run within a transaction.` And openCypher have **no such** requirement of `transaction`.
+
+### How is the time spent value at the end of each return message calculated?
+
+Take the return message of `SHOW SPACES` as an example:
+
+```nGQL
+nebula> SHOW SPACES;
++------+
+| Name |
++------+
+| nba  |
++------+
+Got 1 rows (time spent 1235/1934 us)
+```
+
+* The first number 1235 shows the time spent by the database itself, that is, the time it takes for the query engine to receive a query from the client, fetch the data from the storage server and perform a series of calculations.
+
+* The second number 1934 shows the time spent from the client's perspective, that is, the time it takes for the client from sending a request, receiving a response, and displaying the result on the screen.
+
+### `[ERROR (-7)]: SyntaxError: syntax error near ` '`  
+
+In most cases, a query text requires a `YIELD` or a `RETURN`.
```

Comment:
What is this? I don't understand. An explanation for an error message?

Reply:
This is a common pitfall when no `YIELD` or `RETURN` is provided.